### PR TITLE
CEDS-1466 CEDS-1249 Changed SealSpec TransportDetailsControllerSpec

### PR DESCRIPTION
**Pull request by PatrykRudnicki**

### CODE REVIEW

File: test/unit/forms/declaration/SealSpec.scala [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+  val form = Seal.form
+
+  "Seal object" should {
+
+    "has correct form id" in {
+
+      Seal.formId must be("Seal")
+    }
+
+    "has correct seals limit" in {
```
Comment (bambuchaAdm) [inline test/unit/forms/declaration/SealSpec.scala]: i would put limit into test name

File: test/unit/forms/declaration/SealSpec.scala [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+    }
+
+    "has errors" when {
+
+      "id field is empty" in {
+
+        val emptyForm = Map("id" -> "")
+
+        val result = form.bind(emptyForm)
+        val error = result.errors.head
```
Comment (bambuchaAdm) [inline test/unit/forms/declaration/SealSpec.scala]: what is it contains more errors ?

File: test/unit/forms/declaration/SealSpec.scala (hunk 1 above)
Comment (PatrykRudnicki) [reply test/unit/forms/declaration/SealSpec.scala]: Noted, will change it in next PR

File: test/unit/forms/declaration/SealSpec.scala [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
+    "has no errors" when {
+
+      "id field contains correct value" in {
+
+        val correctForm = Seal("12345")
+
+        val result = form.fill(correctForm)
+
+        result.hasErrors must be(false)
```
[the remark this answers is not in the document]
Comment (PatrykRudnicki) [reply test/unit/forms/declaration/SealSpec.scala]: You're right, is `result.errors must be(Seq.empty)` better?

File: test/unit/forms/declaration/SealSpec.scala (hunk 2 above)
Comment (PatrykRudnicki) [reply test/unit/forms/declaration/SealSpec.scala]: It's only one field, it's not possibile to have more. But I can change this to compare `result.errors` with Sequence with this specific error or add case that length of errors must be 1

File: test/unit/forms/declaration/SealSpec.scala (hunk 3 above)
[the remark this answers is not in the document]
Comment (bambuchaAdm) [reply test/unit/forms/declaration/SealSpec.scala]: must be (empty)

File: test/unit/forms/declaration/SealSpec.scala (hunk 2 above)
Comment (bambuchaAdm) [reply test/unit/forms/declaration/SealSpec.scala]: We would have much more validation of errors - maybe some custom matcher would be nice

**Pull request merged**